Protecs/src/test/java/uk/ac/ucl/protecs/helperFunctions/helperFunctions.java

```diff
@@ -0,0 +1,33 @@
+package uk.ac.ucl.protecs.helperFunctions;
+
+import java.util.ArrayList;
+import java.util.Random;
+import uk.ac.ucl.protecs.sim.Params;
+import uk.ac.ucl.protecs.sim.WorldBankCovid19Sim;
+
+public class helperFunctions {
+	
+	public static WorldBankCovid19Sim CreateDummySim(String paramsFilename, boolean demography) {
+		Random rand = new Random();
+		int seed = rand.nextInt(100000000);
+		System.out.println("Running with seed = " + String.valueOf(seed));
+		Params p = new Params(paramsFilename, false);
+		WorldBankCovid19Sim myWorld = new WorldBankCovid19Sim(seed, p, "", demography);
+		return myWorld;
+	}
+	public static WorldBankCovid19Sim CreateDummySim(int seed, String paramsFilename, boolean demography) {
```

Comment:
I can abstract this a little :)

Reply:
If don't have time to do this, you could describe what you'd like done here and I can do this or attempt to

Reply:
Discussed during call - I will commit the version from our call